mapper/mapper.go

```diff
@@ -21,9 +21,10 @@ func AllDatasets(datasets dataset.List) []model.Dataset {
 
 	sort.Slice(mappedDatasets, func(i, j int) bool {
 		if mappedDatasets[i].Title == "" {
-			return false
-		} else if mappedDatasets[j].Title == "" {
-			return true
+			mappedDatasets[i].Title = mappedDatasets[i].ID
```

Comment:
Bear in mind that you are modifying the struct in the map here by setting the title to the ID. This might have unintended consequences.

It might be cleaner to have something like:
```
iVal = i.Title
if iVal == "" {
  iVal.ID
}

jVal = j.Title
if jVal == "" {
  jVal.ID
}
return iVal < jVal
```

That way you are not modifying the structs and its a bit clearer that you are using a default value to sort on.